var/spack/repos/builtin/packages/py-nbconvert/package.py

```diff
@@ -13,21 +13,37 @@ class PyNbconvert(PythonPackage):
     homepage = "https://github.com/jupyter/nbconvert"
     url      = "https://github.com/jupyter/nbconvert/archive/4.2.0.tar.gz"
 
+    version('5.6.0', sha256='21a69fe5756a3af8e76dcc105217cc3bea4d8f23dbff5727d9fd1e4388b53beb')
     version('4.2.0', sha256='32394be5a20f39f99fabfb9b2f2625df17f1c2a6699182ca41598e98e7cc9610')
     version('4.1.0', sha256='459f23381411fd1ff9ec5ed71fcd56b8c080d97b3a1e47dae1c5c391f9a47266')
     version('4.0.0', sha256='00e25eeca90523ba6b774b289073631ef5ac65bb2de9774e9b7f29604516265c')
 
+    variant('execute', default=True)
+    variant('serve', default=True)
+
     depends_on('py-pycurl', type='build')
```

Comment:
This dependency might only be needed for old versions of Python. Can you try:
```suggestion
    depends_on('py-pycurl', when='^python@:2.7.8', type='build')
```
and see if you can build without it on newer versions?

Reply:
Still builds with python@3.7.4, I think you are correct